orchestration/dags/data_gcp_dbt/models/analytics/diversification/diversification_raw.sql

```diff
@@ -95,7 +95,7 @@ LEFT JOIN {{ source('clean','subcategories') }} subcategories
   ON offer_metadata.subcategory_id = subcategories.id
 LEFT JOIN {{ref('item_clusters_topics_labels') }} as ictl
   ON bookings.item_id = ictl.item_id
-LEFT JOIN {{source('analytics','titelive_gtl_mapping') }} as gtlmap
+LEFT JOIN {{ ref('applicative_titelive_gtl') }} as gtlmap
```

Comment:
```suggestion
LEFT JOIN {{ ref('int_applicative__titelive_gtl') }} as gtlmap
```
Je pense que c'est ce modèle auquel tu veux faire référence? je ne trouve pas de modèle qui s'appelle `applicative_titelive_gtl`